YoYoProject/Controllers/GMSprite.cs

```diff
@@ -336,9 +336,12 @@ internal override void Deserialize(ModelBase model)
 
         internal override void FinalizeDeserialization(ModelBase model)
         {
-            // TODO Implement
-            //TextureGroup = Project.Resources.Get<GMMainOptions>()
-            //                      .Graphics.GetTextureGroup(sprite.textureGroupId);
+            var spriteModel = (GMSpriteModel)model;
+            var tgGuid = spriteModel.textureGroupId;
```

Comment:
Bad variable name. Rename "textureGroupId", or "tgId". The fact that it's a "guid" is irrelevant info in this context.